Clustering/KMeans/src/main/java/org/tribuo/clustering/kmeans/KMeansTrainer.java

```diff
@@ -215,12 +220,7 @@ public KMeansTrainer(int centroids, int iterations, DistanceType distType, int n
      */
     @Deprecated
     public KMeansTrainer(int centroids, int iterations, Distance distanceType, Initialisation initialisationType, int numThreads, long seed) {
-        this.centroids = centroids;
-        this.iterations = iterations;
-        this.distType = distanceType.getDistanceType();
-        this.initialisationType = initialisationType;
-        this.numThreads = numThreads;
-        this.seed = seed;
+        this(centroids, iterations, distanceType.getDistanceType(), initialisationType, numThreads, seed);
```

Comment:
The base constructor already calls postConfig, so this one doesn't need to.

Reply:
indeed, missed that.